test/simple_recipes.hpp

```diff
@@ -117,7 +117,11 @@ class cable1d_recipe: public simple_recipe_base {
     }
 
     cell_size_type num_targets(cell_gid_type i) const override {
-        return cells_.at(i).synapses().size();
+        cell_size_type total_synapses = 0;
```

Comment:
Here you can use `util::sum_by` to avoid the for loop.

Reply:
Right, I always forget what we have in `util`

Reply:
Me too!